src/SIL.XForge.Scripture/ClientApp/src/assets/i18n/non_checking_en.json

```diff
@@ -320,19 +319,33 @@
     "scripture_reference": "Scripture reference"
   },
   "roles": {
+    "allow_add_edit_questions": "Can add & edit questions",
+    "allow_manage_audio": "Can manage scripture audio",
+    "cancel": "Cancel",
+    "learn_more": "Learn More",
     "none": "None",
+    "permissions": "Additional Permissions",
     "pt_administrator": "Administrator",
     "pt_consultant": "Consultant/Reviewer/Archivist/Typesetter",
     "pt_observer": "Observer",
     "pt_translator": "Translator",
+    "roles": "Roles",
+    "roles_from_pt_cannot_be_changed": "Roles can only be changed in Paratext",
+    "save": "Save",
     "sf_community_checker": "Community Checker",
     "sf_commenter": "Commenter",
-    "sf_observer": "Viewer"
+    "sf_observer": "Viewer",
+    "user_pt": "Paratext User",
+    "user_sf": "Scripture Forge User"
   },
   "role_descriptions": {
-    "sf_community_checker": "Answer questions, Comment on answers",
-    "sf_commenter": "View translation, Add comments",
-    "sf_observer": "View translation"
+    "pt_administrator": "Edit translations - manage notes and questions - manage users and settings - sync with Paratext",
```

Comment:
@josephmyers When you made this change, you swapped out commas for hyphens. However, even your new code in this PR calls `i18nService.localizeRoleDescription('...')`, which expects commas, which it then replaces with a bullet. 

https://github.com/sillsdev/web-xforge/blob/cb81cce74820a3802cdc34b140acb443043e6026/src/SIL.XForge.Scripture/ClientApp/src/xforge-common/i18n.service.ts#L229-L231

I originally thought your change was an intentional stylistic choice to use hyphens instead of bullets, but now I see you didn't update the I18nService. Can you either update it to replace hyphens with bullets, or remove the replacing altogether?

Reply:
Sure, I missed this back then. I think the bullets look slightly flashier
https://github.com/sillsdev/web-xforge/pull/2744